x-pack/plugins/observability/public/application/index.tsx

```diff
@@ -98,11 +97,9 @@ export const renderApp = ({
               <EuiThemeProvider darkMode={isDarkMode}>
                 <i18nCore.Context>
                   <RedirectAppLinks application={core.application} className={APP_WRAPPER_CLASS}>
-                    <DatePickerContextProvider>
```

Comment:
💬 This context was responsible for adding extra query parameters to the alerts page and messing with history.